modules/040-node-manager/templates/node-group/service-account.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: ServiceAccount
```

Comment:
* Rename this file to the `rbac-for-us.yaml`
* Change destination namespaces to the `d8-cloud-instance-manager`
* Change its name

This will fix the problem with the linter.

Reply:
Namespace changed to d8-system because of other lint rules.

Reply:
Lint rules allow you to deploy it to the `d8-cloud-instance-manager` namespace. You only need to give the sa a different name.

Reply:
Done